src/keybindings.c

```diff
@@ -124,7 +124,10 @@ GdkModifierType keybindings_get_modifiers(GdkModifierType mods)
 {
 #ifdef __APPLE__
 	if (mods & GDK_MOD2_MASK)
+	{
 		mods |= GEANY_PRIMARY_MOD_MASK;
+		mods &= ~GDK_MOD2_MASK;
```

Comment:
You could do (not sure if it's better though)
```c
mods = mods ^ GDK_MOD2_MASK | GEANY_PRIMARY_MOD_MASK;
```

Reply:
I slightly prefer my version - it's a bit easier to understand (at least by my poor brain) and it's guaranteed to set/clear the bits while the XOR depends on the actual value of mods (but it probably won't contain GEANY_PRIMARY_MOD_MASK bit and is guaranteed to have GDK_MOD2_MASK set).

Reply:
Unless I'm mistaken, in the expression above `^` has priority over `|` so it's the same as `(mods ^ GDK_MOD2_MASK) | GEANY_PRIMARY_MOD_MASK`.
But the simple fact you said what you did makes your version better, as it's then less confusing.

Reply:
Yep, the original version is better so dummies like me who don't know priorities don't get confused :-).